array-manipulation-2/src/includes.js

```diff
@@ -0,0 +1,18 @@
+/* exported includes */
+
+function includes(array, value) {
+
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      return true;
+    }
+  }
+
+  if (i === array.length) {
+    return false;
+  }
+
+  if (i === 0) {
+    return false;
+  }
```

Comment:
You can just `return false` out here, that is the only work you might want to do.